app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -25,20 +25,22 @@ import { CoinList, Coin } from '../common/types';
 
 const STATUSES = {
   LOADING: 'Loading',
+  INFINITE_SCROLL_LOADING: 'InfiniteScrollLoading',
```

Comment:
FWIW I usually mirror values, e.g. `INFINITE_SCROLL_LOADING: 'INFINITE_SCROLL_LOADING'` but not critical as well.